boards/st/nucleo_n657x0_q/twister.yaml

```diff
@@ -0,0 +1,16 @@
+name: ST Nucleo N657X0-Q
+type: mcu
+arch: arm
+ram: 1024
+flash: 1024
+supported:
+  - gpio
+  - uart
+vendor: st
+variants:
```

Comment:
non-blocking: not really familiar with this way of describing the variants but wondering if having separate yaml files isn't better for consistency with the 99.9% of the other boards in tree

Reply:
I found this handy to avoid duplicating information. It seems to have been introduced for this purpose (See https://github.com/zephyrproject-rtos/zephyr/pull/77250)
And also I didn't manage to exclude one variant from twister using 2 separate yaml files.